modules/bmo-about-the-bare-metal-operator.adoc

```diff
@@ -0,0 +1,69 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_module-type: CONCEPT
+[id="bmo-about-the-bare-metal-operator_{context}"]
+= About the Bare Metal Operator
+
+Use the Bare Metal Operator (BMO) to provision, manage, and inspect bare-metal hosts in your cluster. 
+ 
+The BMO uses three resources to complete these tasks: 
+
+* `BareMetalHost`
+* `HostFirmwareSettings`
+* `FirmwareSchema`
+
+The BMO maintains an inventory of the physical hosts in the cluster by mapping each bare-metal host to an instance of the `BareMetalHost` custom resource definition. Each `BareMetalHost` resource has hardware, software, and firmware details specific to a bare-metal host in your cluster. 
+
+The BMO interfaces with bare-metal hosts in the cluster by using Metal3's Ironic service. The Ironic service uses the Baseboard Management Controller (BMC) on the host to interface with the machine. The BMO continually inspects the bare-metal hosts in your cluster to ensure each `BareMetalHost` resource accurately reflects the hardware, software, and firmware details for each bare-metal host in the cluster. 
```

Comment:
"Metal3's Ironic service" is a confusing way to put it. Ironic is not developed as part of Metal3 (BMO is). Maybe a bit better to say "by using the Ironic service as a backend" or something like that.

It may be worth including a link https://ironicbaremetal.org/

If you want to mention Metal3 (the community would be grateful for that - we get little exposure), line 9 is probably a better place since BMO is one of the main deliverables of the Metal3 project.

Reply:
I changed from "Metal3's Ironic service" to the "Ironic API service" in the first instance in the doc, then I refer to just the "Ironic service" in subsequent instances. Hope that works and is accurate. 

I will include the Ironic bare metal link you provided in the **Additional resoures** section. I will also include a link to metal3.io. I think that's probably the best place to mention Metal3 as users can continue there for further reading, git gives a mention for the community, and it doesn't distract users from the main body of the content.